src/c65_startup.hpp

```diff
@@ -15,22 +15,24 @@
 
 void setControlMode(bool); // In CTRL menu
 
-#define StartupIsInHomePosition 1
-#define StartupSetRoll 2
-#define StartupWaitForRollCompletion 3
-#define StartupRollConfirmed 4
-#define StartupSetHATime 10
-#define StartupWaitForHACompletion 15
-#define StartupHAConfirmed 20
-#define StartupWaitForPoleCompletion 25
-#define StartupPoleConfirmed 30
-#define StartupCompleted 35
+enum startupState_t {
+    StartupIsInHomePosition = 1,
```

Comment:
@andre-stefanov Is there meaning to the numbering scheme here? If not we could just drop the assignment in the enumeration

Reply:
You can drop the numbers